test/commit-and-push.ts

```diff
@@ -37,6 +44,16 @@ type CreateCommitResponse = GetResponseTypeFromEndpointMethod<
   typeof octokit.git.createCommit
 >;
 
+class FakeCommitSigner implements CommitSigner {
+  signature: string;
+  constructor(signature: string) {
+    this.signature = signature;
+  }
+  async generateSignature(_commit: CommitData): Promise<string> {
+    return this.signature;
```

Comment:
I would add an ignore line, to avoid this warning.

Reply:
Done.